docs/website/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "renovate-docs",
+  "version": "0.0.0",
+  "private": true,
+  "scripts": {
+    "docusaurus": "docusaurus",
+    "start": "run-s compile && docusaurus start",
+    "build": "run-s compile && docusaurus build",
+    "compile": "run-s clean generate",
+    "swizzle": "docusaurus swizzle",
+    "deploy": "docusaurus deploy",
+    "serve": "docusaurus serve",
+    "clean": " git clean -fX -- ./docs/ ./static/",
+    "generate": "run-s generate:*",
+    "generate:files": "node ../../tools/generate-docs.js"
+  },
+  "dependencies": {
+    "@docusaurus/core": "2.0.0-alpha.70",
+    "@docusaurus/preset-classic": "2.0.0-alpha.70",
+    "@mdx-js/react": "1.6.22",
+    "clsx": "1.1.1",
+    "fs-extra": "9.0.1",
+    "react": "17.0.1",
+    "react-dom": "17.0.1",
```

Comment:
Docusaurus v2 is still using React 16. I think you can use React 17, but from memory the Docusaurus build system was complaining about the different versions, so I reverted the upgrade to be on the safe side.

I raised a issue with Docusaurus to to notify them of the update to React 17 (see `#3617` at the `facebook/docusaurus` repository.